python/llama-index-server/examples/hitl/main.py

```diff
@@ -0,0 +1,101 @@
+import hashlib
+import subprocess
+from typing import Type
+
+from fastapi import FastAPI
+from pydantic import BaseModel, Field
+
+# Uncomment this to use the custom workflow
+# from custom_workflow import CLIWorkflow
+from llama_index.core.agent.workflow import AgentWorkflow
+from llama_index.core.workflow import Context, Event
+from llama_index.llms.openai import OpenAI
+from llama_index.server import LlamaIndexServer, UIConfig
+from llama_index.server.api.models import HumanInputEvent
+
+
+class CLIHumanResponseEvent(Event):
```

Comment:
confusing that they don't seem to share events as the custom_workflow - maybe two example folders are better? one for custom one for agent workflow?